### PR TITLE
Docs: Add time series annotations docs

### PR DESCRIPTION
**Pull request by dprokop**
Images PR: https://github.com/grafana/website/pull/5173

### CODE REVIEW

File: docs/sources/panels/visualizations/time-series/annotate-time-series.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Annotate time series
+
+This section explains how to create annotations in the Time series panel. To read more about annotations support in Grafana please refer to [Annotations]({{< relref "../../../dashboards/annotations.md" >}}).
+
+## Add annotation
+
+1. In the dashboard click on the Time series panel. A context menu will appear.
+1. In the context menu click on **Add annotation**.
+   ![Add annotation context menu](/static/img/docs/time-series-panel/time-series-annotations-context-menu.png)
+1. Add annotation description and tags(optional).
```
Comment (ashharrison90) [inline docs/sources/panels/visualizations/time-series/annotate-time-series.md:16]: maybe 'Add **an** annotation description...' instead

File: docs/sources/panels/visualizations/time-series/annotate-time-series.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-annotation.png)
+1. Click save.
+
+Alternatively, to add annotation, you can Ctrl/Cmd+Click on the Time series panel and the Add annotations popover will appear
```
Comment (ashharrison90) [inline docs/sources/panels/visualizations/time-series/annotate-time-series.md:20]: to add annotation -> to add **an** annotation?

File: docs/sources/panels/visualizations/time-series/annotate-time-series.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-annotation.png)
+1. Click save.
+
+Alternatively, to add annotation, you can Ctrl/Cmd+Click on the Time series panel and the Add annotations popover will appear 
+
+## Add region annotation
+
+1. In the dashboard Ctrl/Cmd+click and drag on the Time series panel.
+   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-region-annotation.gif)
+1. Add annotation description and tags(optional).
```
Comment (ashharrison90) [inline docs/sources/panels/visualizations/time-series/annotate-time-series.md:26]: add **an** annotation description...

File: docs/sources/panels/visualizations/time-series/annotate-time-series.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-add-region-annotation.gif)
+1. Add annotation description and tags(optional).   
+1. Click save.
+
+## Edit annotation
+
+1. In the dashboard hover over an annotation indicator on the Time series panel.
+   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-edit-annotation.gif)
+1. Click on the pencil icon in the annotation tooltip.
+1. Modify description and/or tags.
```
Comment (ashharrison90) [inline docs/sources/panels/visualizations/time-series/annotate-time-series.md:34]: Modify **the** description and/or tags.

File: docs/sources/panels/visualizations/time-series/annotate-time-series.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+1. Click save.
+
+## Delete annotation
+
+1. In the dashboard hover over an annotation indicator on the Time series panel.
+   ![Add annotation popover](/static/img/docs/time-series-panel/time-series-annotations-edit-annotation.gif)
+1. Click on the thrash icon in the annotation tooltip.
```
Comment (ashharrison90) [inline docs/sources/panels/visualizations/time-series/annotate-time-series.md:41]: trash instead of thrash

**Review by ashharrison90 — COMMENTED** [review]
super minor nitpicks, i'm not a technical writer so feel free to ignore 😄

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

I'm not a content writer either but agree with Ash's suggestions.

**Comment by dprokop** [comment]
Thank you @ashharrison90 and @jackw for taking a look at this :)

**Pull request merged**